src/Text/index.jsx

```diff
@@ -24,13 +25,14 @@ const Text = ( {
         className = { buildClassName( className, cssMap, {
             allCaps,
             overflowHidden : overflowIsHidden,
+            lineHeight,
```

Comment:
Please remove this line as well as the `.lineHeight__…` blocks in the CSS (see my [comment here](https://github.com/sociomantic-tsunami/nessie-ui/issues/539#issuecomment-422757899))

Reply:
So in this case we won't need to write as a string `S` or `M`..etc?